packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -177,30 +179,27 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		pendingLocalState?: string,
 	): Promise<{ container: IContainerExperimental; connect: () => void }> {
 		const p = new Deferred();
-		const documentServiceFactory = testObjectProvider.driver.createDocumentServiceFactory();
-
-		// patch document service methods to simulate offline by not resolving until we choose to
-		const boundFn = documentServiceFactory.createDocumentService.bind(documentServiceFactory);
-		documentServiceFactory.createDocumentService = async (...args) => {
-			const docServ = await boundFn(...args);
-			const boundCTDStream = docServ.connectToDeltaStream.bind(docServ);
-			docServ.connectToDeltaStream = async (...args2) => {
-				await p.promise;
-				return boundCTDStream(...args2);
-			};
-			const boundCTDStorage = docServ.connectToDeltaStorage.bind(docServ);
-			docServ.connectToDeltaStorage = async (...args2) => {
-				await p.promise;
-				return boundCTDStorage(...args2);
-			};
-			const boundCTStorage = docServ.connectToStorage.bind(docServ);
-			docServ.connectToStorage = async (...args2) => {
-				await p.promise;
-				return boundCTStorage(...args2);
-			};
-
-			return docServ;
-		};
+		const documentServiceFactory = wrapObjectAndOverride<IDocumentServiceFactory>(
+			provider.documentServiceFactory,
+			{
+				createDocumentService: {
+					connectToDeltaStream: (_ds) => async (client) => {
```

Comment:
nit: underscore is usually only used to prefix unused parameters. used parameters should have it

Reply:
You meant to say "used parameters should not have it" right?